src/sql/parts/modelComponents/editor.component.ts

```diff
@@ -108,8 +110,9 @@ export default class EditorComponent extends ComponentBase implements IComponent
 		let width: number = this.convertSizeToNumber(this.width);
 
 		let height: number = this.convertSizeToNumber(this.height);
-		if (this._isAutoResizable) {
-			height = this._editor.scrollHeight;
+		if (this._isAutoResizable && this._minimumEditorSize !== undefined) {
```

Comment:
minimum size should be optional, correct? Shouldn't we always do this if `isAutoResizable`, then use a floor of 0 if `minimumEditorSize` is undefined?